src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -475,6 +494,17 @@ private void Archive(Tournament tournament)
             _tournamentRepository.Update(tournament);
         }
 
+        /// <summary>
```

Comment:
No need to comment private methods

Reply:
No need doesn't mean forbidden, am I right?

Reply:
Yes, so you don't remove existing ones)

Reply:
I don't seem to understand what commentary was removed.

Reply:
Deprecated now.